docker-compose.yml

```diff
@@ -226,8 +226,8 @@ services:
       - elasticsearch
     env_file: .env
     environment:
-      ELASTICSEARCH_URL: http://elasticsearch:9200
-      ELASTICSEARCH_HOSTS: http://elasticsearch:9200
+      ELASTICSEARCH_URL: https://elasticsearch:9200
```

Comment:
Interesting, why?

Reply:
I think Kibana was failing with ssl errors until I did this.

Reply:
I'm gonna revert this. I'm not sure why but it's not working on my machine either way (`http` or `https`). It starts up, but I can't access it at `localhost:5601`